Source/Houdini/Houdini.cs

```diff
@@ -408,19 +409,22 @@ public HoudiniState CurrentHoudiniState
     }
 
     public static TextWriter explainHoudiniDottyFile;
+    private CancellationToken cancellationToken;
 
     protected Houdini()
     {
+      cancellationToken = CancellationToken.None;
     }
 
     public Houdini(Program program, HoudiniSession.HoudiniStatistics stats, string cexTraceFile = "houdiniCexTrace.txt")
     {
       this.program = program;
       this.cexTraceFile = cexTraceFile;
-      Initialize(program, stats);
+      Initialize(program, stats, new CancellationToken());
```

Comment:
How do you decide between `new CancellationToken` and `CancellationToken.None`?

Reply:
I should use `CancellationToken.None` here, I discovered it later.